src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,108 @@
+package duke;
+
+import duke.dukeexception.NoListException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.FileWriter;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Represents a wrapper class that deals with loading and saving tasks in a file.
+ */
+public class Storage {
+    private File file;
+
+    /**
+     * Constructs a Storage from a given filePath.
+     *
+     * @param filePath The given String filePath.
+     */
+    public Storage(String filePath) {
+        this.file = new File(filePath);
+        if (!this.file.getParentFile().exists()) {
+            this.file.getParentFile().mkdirs();
+        }
+    }
+
+    /**
+     * Converts a string that contains info about a Task into a Task.
+     *
+     * @param fullString The given string
+     * @return The Task with the appropriate information.
+     */
+    public static Task stringToTask(String fullString) {
```

Comment:
Perhaps it is better to name this method ```convertToTask()``` such that the method name starts with a verb
```suggestion
    public static Task convertToTask(String fullString) {
```

Reply:
I agree with jhlee1997 that convertToTask might be better as a method name